src/main/java/org/jacp/repository/QuestionRepository.java

```diff
@@ -14,4 +16,7 @@
 public interface QuestionRepository extends BaseRepository<QuestionEntity> {
     @Override
     Optional<QuestionEntity> findById(Long id);
+
+    @Query("from QuestionEntity qe where qe.id in (:id)")
+    List<QuestionEntity> getTasks(List<Long> id);
```

Comment:
Minor: List<Long> ids

Reply:
It also can be without manually written query, like: 
```
List<QuestionEntity> findByIdIn(Collection<Long> ids);
```
Please take a look: https://docs.spring.io/spring-data/jpa/reference/jpa/query-methods.html#jpa.query-methods.query-creation

Reply:
Fixed